Makefile.am

```diff
@@ -14,3 +15,14 @@ coverage: check
 	@echo -e "libica coverage";
 	@echo -e "---------------\n";
 	cd ${top_builddir}/src && gcov .libs/*.gcda
+
+if ICA_FIPS
+install-data-hook:
+	$(INSTALL) -m 0444 ${top_builddir}/src/.libs/.libica.so.$(VERSION).hmac $(libdir)
```

Comment:
Please make the build relocatable.  The install-folder should read `$(DESTDIR)$(libdir)`.  This is typically needed to build RPMs and otherwise has to be patched by distributors.